waiter/src/waiter/token.clj

```diff
@@ -330,7 +330,7 @@
     (when (some #(= token %) waiter-hostnames)
       (throw (ex-info "Token name is reserved" {:status 403 :token token})))
     (when-not (re-matches valid-token-re token)
-      (throw (ex-info "Token must match pattern"
+      (throw (ex-info "Token must be two or more characters, contain alphanumeric, '-', '_', '$', or '.' characters and must start with a letter"
```

Comment:
How do you like:
`"Token must be two or more characters, must start with a letter and contain letters, digits, '-', or '.'"`

Reply:
Also, this change is not relevant to this PR.

Reply:
Something for another PR, we need to improve our token regex, currently it is legl to use `a..-_...`.
Created #483.